kubernetes/tmp/compatibility_matrix.txt

```diff
@@ -6,4 +6,4 @@ config-service:0.1.8
 hypertrace-graphql:0.7.9
 hypertrace-ui:0.70.2
 hypertrace-ingester:0.6.27
-hypertrace-alert-engine:2.0.0
+hypertrace-alert-engine:1.0.0
```

Comment:
why are going back to prior version?

Reply:
to rerun the publish flow so that release notes are correctly filled